clients/command/src/main/java/org/eclipse/hono/client/command/CommandHandlerWrapper.java

```diff
@@ -42,15 +44,21 @@
      *                  parameter.)
      * @param commandHandler The command handler.
      * @param context The vert.x context to run the handler on or {@code null} to invoke the handler directly.
+     * @param consumerCreationSpanContext The span context of the command consumer creation operation to reference as
+     *                                    <em>follow-from</em> span in the command handling spans or {@code null} if
+     *                                    that context is not available or shouldn't be used. Note that providing the
+     *                                    context here only makes sense for short-lived command consumers.
```

Comment:
how about
> The span for tracking the creation of the command consumer that this wrapper belongs to.
If not {@code null}, then the spans created by this wrapper for handling command request messages will have a <em>follows-from</em> reference to the given span. Note that providing a span context here only makes sense for short-lived command consumers as created by the HTTP and CoAP adapters for example.

Reply:
I've adopted parts and rephrased others.